template.sh

```diff
@@ -6,10 +6,13 @@ for i in aws azure common vmware; do
 
   cp -a policies/$i/. helm/policies-$i/templates
 
-  for filename in helm/policies-$i/templates/*.yaml; do
-    sed -i 's/{{/üüü/g' $filename # replace {{ with placeholder
-    sed -i 's/}}/{{ `}}` }}/g' $filename # replace }} with correct helm substitution
-    sed -i 's/üüü/{{ `{{` }}/g' $filename # replace the placeholder correct helm substition for {{
+  for filename in $(find "helm/policies-$i/templates" -name "*.yaml"); do
+    if [[ $(basename $filename) == "Pod.yaml" ]]; then
```

Comment:
I don't want to escape `{{` or `}}` in this file because I want it to be templated by helm. Open to suggestions if someone sees a better way. Maybe tag lines with a comment if they should be skipped?

Reply:
My idea (not implemented yet) was to use `[[` and `]]` which would then translate to the helm delimeters. 

There is an open kyverno issue to change the delimeter or make it configurable.

Reply:
That sounds good. I'll give it a try.

Reply:
All done.